Userland/Libraries/LibJS/Runtime/DatePrototype.cpp

```diff
@@ -178,7 +178,7 @@ JS_DEFINE_NATIVE_FUNCTION(DatePrototype::set_full_year)
     datetime.set_time(new_year, new_month, new_day, datetime.hour(), datetime.minute(), datetime.second());
     this_object->set_is_invalid(false);
 
-    return Value { this_object->time() };
+    return Value(this_object->time());
```

Comment:
:eyes:

Reply:
for the sake of consistencyyyyy

(only use of `Value { foo }` in all of LibJS, can't have that)

Reply:
Yeah yeah all right then Linus 😄